4.analyze_data/notebooks/linear_model/nbconverted/0.fit_linear_model_strat.py

```diff
@@ -38,13 +44,13 @@
 # In[3]:
 
 
-# Filter by failing hearts and specific treatments
-specific_type = ["DMSO"]
-specific_cell_types = ["failing", "healthy"]
+# Filter by cell type and only cells without DMSO treatment
+specific_type = [7]
+# specific_cell_types = ["Failing", "Healthy"]
 
 filtered_df = data_df[
-    (data_df['Metadata_treatment'].isin(specific_type)) &
-    (data_df['Metadata_cell_type'].isin(specific_cell_types))
+    (data_df['Metadata_heart_number'].isin(specific_type))
+    # (data_df['Metadata_cell_type'].isin(specific_cell_types))
```

Comment:
Consider removing commented code if you plan on no longer using it.

Reply:
Just changed this when re-running the code, thanks!